corehq/apps/es/app_config.py

```diff
@@ -5,32 +5,6 @@ class ElasticAppConfig(AppConfig):
 
     name = 'corehq.apps.es'
 
-    def __init__(self, *args, **kw):
```

Comment:
I might have missed something in previous commits but can you please explain why we would not need to purge tombstones on startup?

Reply:
Since the multiplexer now creates tombstones _only when the secondary delete fails_, new tombstones will not get created once they primary and secondary are completely synchronized (reindexed). This means that the when the reindex task completes, the multiplexer will no longer be creating new tombstones, and the post-sync tombstone cleanup step will permanently clear all tombstones. I hope the [README changes](https://github.com/dimagi/commcare-hq/pull/32644/files#diff-9219c40f157c25b29ee884e59e78bb75b2b010db5b25a507dcff9223079e0062) outline this clearly, if not I'd be happy to offline on how to improve it.

Reply:
That's awesome. Thanks for explaining.